src/main/java/com/google/sps/servlets/HistoryServlet.java

```diff
@@ -48,7 +68,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
         List<TutorSession> scheduledSessions = new ArrayList<TutorSession>();
 
-        for (Student student : SampleData.getSampleStudents()) {
+        for (Student student : datastore.getStudents()) {
```

Comment:
I'll replace this with the datastore implementation after merging.

Reply:
Perfect, thanks!